Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,10 +1,170 @@
+* Added a filter to remove any invisible characters introduced by Zoom when copy-pasting room names, IP addresses, or ports. 
+* Boss Room now uses the [UnityToonShader](https://github.com/IronWarrior/UnityToonShader) for rendering the 3D surfaces to emulate 2D, flat surfaces.
+* Added disconnection error message to load when a player or host disconnects due to limited or no network connectivity. Client logic was also updated to detect Host disconnection scenarios, such as losting connectivity.
+* Balanced hero and enemy stats, spawners, trigger areas, and enemy detetction areas. 
+* Added healthbars to display when damaged and properly track imp health locally and across clients. 
+
+### Changes
+
+* Updated Boss Room per asset and project settings for increased performance and better resource management:
+
+  * Disabled GPU Skinning to optimize GPU usage and FPS.
+  * Lowered quality of ambient occlusion from high to medium.
+  * Switched SMAA High to FXAA (fast mode) reducing GPU cost.
+  * Modified GPU Instancing on imps, heroes, and the boss to significantly reduce the number of draw calls.
+  * Turned off Cast Shadows on Imp and Imp Boss.
+  * Disabled mesh colliders of lava, which is more decorative than interactive.
+  * Refactored the S_SimpleDissolve shader which consumed most import time.
+
+* Updated the Photon Setup Guide, indicating you need only app ID when playing with friends. For users connecting across regions, you may need to hard code a region in your app settings by using the room code and region instead of just the room code sharing in game. 
+* Removed a duplicated `GameObject` from the MainMenu scene. 
+* Reviewed and revised code to better following quality standards.
+* Updated the Mage character base attack to better support the new enqueuing ability and handle game behaviors. Updates include:
+
+  * Actions being non-blocking now allow other actions while a mage-bolt is in flight
+  * Actions send in `ActionSequence` groups to better handle actions when players spam click enemies
+  * Timing issues with animations, actions, and character location
+  * Bolt animation for legitimate hits
+  * Updated attacks not to knock rogues out of stealth 
+
+* Updated character attacks to not cause friendly-fire damage between players. 
+* Updated and resolved issues with 3D models including polygon count of coins and chest, artifacts where level graphics are stitched together or overlapping characters, and asset map consistency used by objects (color + normal map + emission).
+* Merged `ConnectStatus` and `DisconnectReason` into a single `ConnectStatus`. 
+* Updated `ServerGameNetPortal` to properly handle the following:
+
+  * Per-connection state on client disconnect
+  * Additional errors including a full server and player ID (GUID) already playing
+  * Player connection if they experience a game-level connection failure
```

Comment:
This is a duplicate from a previous entry saying we handle connection failures

Reply:
Pulling thanks!
```suggestion

```